requirements/requirements.dev.txt

```diff
@@ -2,99 +2,100 @@
 # This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
-#    pip-compile requirements.dev.in
+#    pip-compile --strip-extras requirements.dev.in
```

Comment:
Note on `--strip-extras` - this will be the default behaviour starting from `pip-tools==8.0.0`. In our case, I added it in order to correctly parse the `psycopg` related dependencies.